mobile/src/main/java/rs/readahead/washington/mobile/views/activity/CameraActivity.java

```diff
@@ -344,9 +345,19 @@ public Context getContext() {
 
     @Override
     public void onMediaFilesUploadScheduled() {
-        if (intentMode != IntentMode.STAND) {
-            finish();
+        boolean isAutoUploadEnabled = Preferences.isAutoUploadEnabled();
```

Comment:
maybe we could make the code more reusable using a method of showing the bottom message that could be reused the MicFragment as well , something like this:                                                                                                                                           
   if (isAutoUploadEnabled && isAutoDeleteEnabled) {
    showMessageDialog(this, getString(R.string.AUTO_UPLOAD_AND_DELETE_MESSAGE));
} else if (isAutoUploadEnabled) {
    showMessageDialog(this, getString(R.string.AUTO_UPLOAD_MESSAGE));
}

private void showMessageDialog(Context context, String message) {
    DialogUtils.showBottomMessage(context, message, false);
}